src/components/Footer/__tests__/Footer.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import { render, fireEvent } from "@testing-library/vue";
+import Footer from "./../";
+
+describe("components/Footer", () => {
+  it("calls handleClick callback on click", async () => {
+    const spy = jest.fn();
+    const copyright = "this is my copyright";
+    const { getByTestId } = render(Footer, {
+      props: {
+        copyright: copyright,
+        links: [
+          {
+            referenceId: 1,
+            referenceType: "page",
+            isActive: true,
+            label: "footer1",
+          },
+          {
+            referenceId: 2,
+            referenceType: "page",
+            isActive: false,
+            label: "footer2",
+          },
+        ],
+        handleClick: spy,
+      },
+    });
+    const footer = getByTestId("footer-1");
+    await fireEvent(footer!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
+  });
+  it("checks if all items have been rendered", async () => {
+    const spy = jest.fn();
+    const copyright = "this is my copyright";
+    const links = [
+      {
+        referenceId: 1,
+        referenceType: "page",
+        isActive: true,
+        label: "footer1",
+      },
+      {
+        referenceId: 2,
+        referenceType: "page",
+        isActive: false,
+        label: "footer2",
+      },
+    ];
+    const { getAllByText } = render(Footer, {
+      props: {
+        copyright: copyright,
+        links: links,
+        handleClick: spy,
+      },
+    });
```

Comment:
Please use the object initializer shorthand:
`props: { copyright, links, handleClick: spy },`